docs/getting-started.md

```diff
@@ -1,4 +1,8 @@
-# Getting Started
+# Getting Started for Users
+
+Coming soon!
+
+# Getting Started For Developers
```

Comment:
Let's add something to the docs here (in the spirit of the docs-diff Github Action)